app/js/lib/foxbox/recipes.js

```diff
@@ -0,0 +1,300 @@
+'use strict';
+
+// @todo Proxy recipes in the db and emit events on change (see services).
+
+// Private members.
+const p = {
+  // Private properties.
+  settings: Symbol('settings'),
+  net: Symbol('net')
+};
+
+export default class Recipe {
+  constructor(props) {
+    this[p.settings] = props.settings;
+    this[p.net] = props.net;
+  }
+
+  /**
+   * Returns a promise resolving to a list of recipes.
+   *
+   * @return {Promise}
+   */
+  getAll() {
+    return this[p.net].fetchJSON(
+      `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
```

Comment:
I want to integrate it directly in `this[p.net].fetchJSON()` so that API calls are less verbose.